cve_bin_tool/data_sources/osv_source.py

```diff
@@ -20,6 +23,17 @@
 from cve_bin_tool.version import HTTP_HEADERS
 
 
+def find_gsutil():
+    gsutil_path = shutil.which("gsutil")
+    if gsutil_path is None:
+        gsutil_path = f"{sys.prefix}/bin/gsutil"
```

Comment:
So, I've been debating the security implications of this one.  I think adding a possible new path that the user hasn't set themselves is probably incorrect behaviour, even if in this case the path is probably not actually a problem and likely was already in the user's path options.  Can you rework this so we're not adding to the search options beyond what `shutil.which` is going to find using the user's existing path settings?